cmd/natsdumper/csvwriter/csv_writer.go

```diff
@@ -0,0 +1,197 @@
+package csvwriter
```

Comment:
i did now move it outside, since want to remove the utils folder i have no clue where to put it